index.js

```diff
@@ -63,6 +68,27 @@ app.get('/get', function (req, res) {
   res.send(getCurrentTeamMember());
 });
 
+// Handling slah commands in Slack
+app.get('/slack', function (req, res) {
+  res.sendStatus(200)
+})
+
+app.post(bodyParser.urlencoded({extended:true}), function(req, res) {
```

Comment:
Does this POST respond on `/slack`? I see the GET above is updated (and still can be removed), but I'd expect the POST to look like `app.post('/slack', bodyParser......)`.

Reply:
It has worked in all of my tests